File: Assets/GraphQL/Model/Trips/Review.cs
using GraphQL.Helpers;

namespace GraphQL.Model {
	public class Review {
		[GqlSelection("reviewID")] public int ReviewID { get; private set; }
		[GqlSelection("tripID")] public int TripID { get; private set; }
		[GqlSelection("rating")] public int Rating { get; private set; }
		[GqlSelection("description")] public string Description { get; private set; }
	}
}

File: Server/src/schema/Trips/review.ts
import { Field, InputType, Int, ObjectType } from "type-graphql";
import { BaseEntity, Column, Entity, PrimaryGeneratedColumn } from "typeorm";

@Entity()
@ObjectType()
export default class Review extends BaseEntity {
    @Field(() => Int!)
    @PrimaryGeneratedColumn()
    reviewID!: number;

    @Field(() => Int!)
    @Column()
    tripID!: number;

    @Field(() => Int!)
    @Column()
    rating!: number;

    @Field(() => String!)
    @Column()
    description!: string;

    constructor() {
        super();

        this.reviewID = 0;
        this.tripID = 0;
        this.rating = 1;
        this.description = "None";
    }
}